resources/lang/en-US/admin/hardware/table.php

```diff
@@ -25,7 +25,6 @@
     'image'		    => 'Device Image',
     'days_without_acceptance' => 'Days Without Acceptance',
     'monthly_depreciation' => 'Monthly Depreciation',
-    'assigned_to'    => 'Assigned To',
```

Comment:
Huh - why did this go away? Is that OK?

Reply:
It was a duplicate strong. My IDE highlight it when I opened the file for a different reason.